app/src/main/java/com/findmyrecycling/MainActivity.kt

```diff
@@ -35,19 +40,29 @@ fun RecycleSearch(name: String) {
     var location by remember{ mutableStateOf("")}
 
     Column {
-        OutlinedTextField(
+        TextField(
             value = recyclable,
             onValueChange = {recyclable = it},
             label = { Text(stringResource(R.string.recyclable)) },
-            modifier = Modifier.fillMaxWidth()
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(start = 4.dp, end = 4.dp, top = 2.dp, bottom = 2.dp)
```

Comment:
The TextFields were running up against the edges of the screen.  Adding padding to the start and end gave a better look and feel to the screen